lib/components/atoms/Input/Input.story.js

```diff
@@ -1,12 +1,13 @@
 // @flow
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-import { textInputElementLocator } from './Input.mock';
+import { textInputElementLocator, fileInput } from './Input.mock';
 
 // Import Vanilla Component to display markup and props
 import Input, { InputVanilla } from '.';
 
 storiesOf('Atoms/Input', module)
   .addParameters({ jest: ['Input'] })
   .add('Knobs', () => <InputVanilla id="username" label="User Name" type="text" name="username" />)
+  .add('File Input', () => <InputVanilla {...fileInput} />)
```

Comment:
Check this as we are using InputVanilla here.

Reply:
Vanilla should only be used on Knobs, because we want to avoid storybook wrapping on component

Reply:
Changed to Input, Not using InputVanila.